introduction.md

```diff
@@ -1,3 +1,6 @@
 introduction of emcek aka mipl
-
-My fav path and code schole is JavaScript and Electives.
+deadly skills
+==============
+* Git
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?